app/controllers/ontologies_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
 
   def show
     @content_object = :ontology
+    params[:id] = params[:ontology_id] if params[:ontology_id]
```

Comment:
`'/ref/:reference/:repository_id/*locid' => 'ontologies#show'` this one. But since this route is not needed, it can be removed.